tests/integration/test_corecli.py

```diff
@@ -30,6 +33,26 @@ def test_extract_core(default_cluster):
     assert pkgInfo['package']['name'] == 'dcos-core-cli'
 
 
+@pytest.mark.skipif(os.environ.get('DCOS_TEST_CORECLI') is None, reason="no core CLI bundle")
+@pytest.mark.skipif(sys.platform == 'win32',
+                    reason='Not yet concurrent-safe on Windows (DCOS_OSS-4843)')
+def test_extract_core_concurrently(default_cluster):
+    code, out, err = exec_cmd(['dcos', 'plugin', 'remove', 'dcos-core-cli'])
+    assert code == 0
+    assert out == ''
+    assert err == ''
+
+    with futures.ThreadPoolExecutor() as pool:
+        cmds = [pool.submit(exec_cmd, ['dcos', 'node']) for _ in range(50)]
```

Comment:
```suggestion
        cmds = [pool.submit(exec_cmd, ['dcos', 'node', 'list']) for _ in range(50)]
```
I would suggest using this line, since `dcos node` is deprecated? Would require a small change to the backport.